packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,116 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+
+/**
+ * A Tooltip Custom HTML Element.
+ * @public
+ */
+export class Tooltip extends FASTElement {
+  public elementInternals = this.attachInternals();
+
+  /**
+   * Set the delay for the tooltip
+   */
+  @attr
+  public delay: number = 250;
+
+  /**
+   * Set the positioning of the tooltip
+   */
+  @attr
+  public positioning: string = 'above';
+
+  /**
+   * Reference to the tooltip element
+   * @internal
+   */
+  public tooltip!: HTMLElement;
+
+  /**
+   * The id of the anchor element for the tooltip
+   * @public
+   */
+  @attr
+  public anchor: string = '';
+
+  /**
+   * Reference to the anchor element
+   * @internal
+   */
+  private anchorElement: HTMLElement | null = null;
```

Comment:
Maybe this can be a getter so you don't have to rely on pre-existing assignments before using it? e.g.

```ts
private anchorElement(): HTMLElement | null {
  return document.getElementById(this.anchor ?? '');
}
```

Reply:
Updated to a getter!